src/modules/user/pre-created-users.repository.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+
+import { User } from './user.domain';
+import { PreCreatedUserModel } from './pre-created-users.model';
+import { PreCreatedUserAttributes } from './pre-created-users.attributes';
+
+@Injectable()
+export class SequelizePreCreatedUsersRepository {
+  constructor(
+    @InjectModel(PreCreatedUserModel)
+    private modelUser: typeof PreCreatedUserModel,
+  ) {}
+
+  async findById(id: number): Promise<PreCreatedUserModel | null> {
+    const user = await this.modelUser.findByPk(id);
```

Comment:
return directly instead of using async - await. Also, we are not converting this to domain